test/functions.coffee

```diff
@@ -150,13 +150,13 @@ test "@-parameters and splats with constructors", ->
   eq b, obj.last
 
 test "destructuring in function definition", ->
-  (([{a: [b], c}]...) ->
+  (({a: [b], c}) ->
```

Comment:
I don't think this is acceptable. There are lots of things in programming languages that people never write, but still need to be supported for consistency.